docs/sia_server_guide.md

```diff
@@ -0,0 +1,75 @@
+SIA Server is required for authenticating existing unmanaged services. For
+managed service inception (e.g., Manhattan), other more convenient
+automated mechanisms get the credentials safely to the service. The SIA
+server provides on-box support for securely getting an identity token
+for a running service that was previously registered in ZMS.
+
+The SIA server requires access to that service's [private key](reg_service_guide.md)
+to sign the token. The server runs as a protected local agent, and the services use Unix domain
+socket interface to request tokens. SIA communicates with ZTS to validate service details.
+
+## Installing SIA
+
+The AWS SIA source is part of the Athenz project and can be found in:
+```
+provider/aws/sia-ec2
+``` 
+Follow the readme for instructions on how to install it.
+
+## Private Key Installation
+
+The SIA Server needs to have access the service's private key so it can
+generate and sign a ServiceToken for the service. Then the service can
+use that ServiceToken to authenticate itself in Athenz and, if
+necessary, obtain a RoleToken to process requests against other Athenz
+protected services.
```

Comment:
Removed the explanation on how to install the private keys using Ykeykey or chef as ykeykey isn't open source and so is the logic done by chef. Text reducted.